ghcide/src/Development/IDE/Plugin/HLS/GhcIde.hs

```diff
@@ -61,7 +63,7 @@ descriptor plId = (defaultPluginDescriptor plId desc)
 
 -- ---------------------------------------------------------------------
 
-hover' :: PluginMethodHandler IdeState Method_TextDocumentHover
-hover' ideState _ HoverParams{..} = do
-    liftIO $ logDebug (ideLogger ideState) "GhcIde.hover entered (ideLogger)" -- AZ
-    hover ideState TextDocumentPositionParams{..}
+hover' :: Recorder (WithPriority Hover.Log) -> PluginMethodHandler IdeState Method_TextDocumentHover
+hover' recorder ideState _ HoverParams{..} = do
+    liftIO $ logWith recorder Debug Hover.LogEnterHover
```

Comment:
this is arguably a silly log. if we're going to do this we should do it generically!

Reply:
Removed!